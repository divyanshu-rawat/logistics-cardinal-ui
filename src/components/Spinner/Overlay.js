import styled from 'styled-components';

const SpinnerOverlay = styled('div')`
  width: 100%;
  height: 100%;
  position: absolute;
  top: 0;
  left: 0;
  background-color: rgba(52, 58, 66, 0.8);
  z-index: ${({ theme }) => theme.zIndex.spinnerOverlay};
`;

export default SpinnerOverlay;
