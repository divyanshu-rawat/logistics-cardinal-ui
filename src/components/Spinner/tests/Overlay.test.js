import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Overlay from '../Overlay';
import Theme from '../../../themes/rooster';

describe('<Overlay />', () => {
  it('should render the component', () => {
    const tree = toJson(shallow(<Overlay theme={Theme} />));
    expect(tree).toMatchSnapshot();
  });
});
