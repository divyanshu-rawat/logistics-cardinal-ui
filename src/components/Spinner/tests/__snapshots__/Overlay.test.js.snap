// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Overlay /> should render the component 1`] = `
<div
  className="sc-bdVaJa gtVEEf"
/>
`;
